### PR TITLE
update the list of contributors for 4.7

### PR DESCRIPTION
**Pull request by yashodha**

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -1,6 +1,64 @@
...
+************************************************
+
+CiviCRM Team - Atif Shaikh, Coleman Watts, David Greenberg, Eileen McNaughton,
+	       Jitendra Purohit, Josh Gowan, Kurund Jalmi,
+               Michael McAndrew, Monish Deb, Rohan Ramesh Katkar, Tim Otten,
+               Yashodha Chaku
+
+AGH Strategies - Andrew Hunt, Tyrell Cook, Nikki Murray
+Backoffice Thinking
+Cambridge University
```
Comment (JKingsnorth) [inline CONTRIBUTORS.txt]: Hi, thanks for this. Would it be OK to change it to:
University of Cambridge - Alex Corr, John Kingsnorth

(then you can remove me from further down the list).

**Comment by eileenmcnaughton** [comment]
@JKingsnorth what say I merge this & then you do a PR for how you want it to look?

**Comment by JKingsnorth** [comment]
That's fine, thanks Eileen :)

**Comment by eileenmcnaughton** [comment]
Done! Go for it

**Pull request merged**